### ISSUE TITLE
Publish stable maven artifacts for current master branch

### ISSUE THREAD

**Issue by NebelNidas** [skylot/jadx#2132]
### Describe your idea

The current solution of relying on snapshots (as described [here](https://github.com/skylot/jadx/issues/2106)) is suboptimal when reproducible builds are desired, which is a blocker for https://github.com/FabricMC/fabric-loom/pull/1073. Would it be possible to start publishing them as dev or alpha builds?

**Comment by skylot** [comment]
Well I just need to release new version :rofl:
Right now, I am working on some docs for upcoming new features (scripts and plugins), but I am terrible at writing, so progress is very slow :cry:

**Comment by NebelNidas** [comment]
I just noticed Sonatype's maven actually hosts different artifacts for each snapshot, so I'd be able to depend on e.g. `io.github.skylot:jadx-core:1.5.0-20240227.173751-9`, which works well enough for our use case.

> progress is very slow

No need to hurry, it's better to take your time so everything's polished when this huge update releases 🙂
I was just asking so we can beta-test the current master branch without having to worry about a changing maven artifact breaking things midway through, but as I noticed, Sonatype already provides that.
